app/build.gradle.kts

```diff
@@ -56,6 +56,7 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+    namespace = "io.github.androidacademyglobal"
```

Comment:
maybe also use `aag` acronym as in `core` module or use a full name in `core` module for consistency :)

Reply:
It was extracted from manifest to fix AGP warning. We can change it with package in separate PR :)

Reply:
:) we need more PRs! :)